firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -172,306 +200,269 @@ const double MIN_PACC = 2.3;              // minumum accumulator pressure to mai
-
-
-int calculateSLADutyCycle(float pre) {
-  int scaled = abs(pre) * 512;
-  int scale =  map(scaled, 0, 1024, SLADutyMin, SLADutyMax);
-  return scale;
-}
-
-
-int calculateSLRDutyCycle(float pre) {
-  int scaled = abs(pre) * 512;
-  int scale =  map(scaled, 0, 1024, SLRDutyMin, SLRDutyMax);
-  return scale;
-}
-
-float pressureToVoltage(int MPa) {
-    return ( MPa + 217.1319446 ) / 505.5662053;
-    // convert MPa pressure to equivalent voltage
-    return MPa;
-}
-
-int voltageToPressure( float voltage) {
-    // convert voltage reading from sensors to pressure in MPa
-    return ( voltage * 505.5662053 ) - 217.1319446;
-}
-
-// convert the ADC reading (which goes from 0 - 1023) to a voltage (0 - 5V):
-float convertToVoltage(int input) {
-    return input * (5.0 / 1023.0);
-}
+FSM brakeStateMachine = FSM( Wait );
 
 
+// *****************************************************
+// local data structures
+// *****************************************************
 // accumulator structure
 struct Accumulator {
     float _pressure = 0.0;    // pressure is initliazed at 0
-    byte _sensorPin = 99;     // set to 99 to avoid and accidental assignments
-    byte _controlPin = 99;
-    Accumulator( byte sensorP, byte relayP );
+    byte _sensor_pin = 99;     // set to 99 to avoid and accidental assignments
+    byte _control_pin = 99;
+    Accumulator( byte sensor_p, byte relay_p );
 
-    void updatePressure()
+    void updatePressure( )
     {
     }
 
-
     // turn relay on or off
-    void pumpOn()
+    void pumpOn( )
     {
-      digitalWrite(_controlPin, HIGH);
+      digitalWrite( _control_pin, HIGH );
     }
 
-
-    void pumpOff()
+    void pumpOff( )
     {
-      digitalWrite(_controlPin, LOW);
+      digitalWrite( _control_pin, LOW );
     }
 
     // maintain accumulator pressure
-    void maintainPressure()
+    void maintainPressure( )
     {
-      _pressure = convertToVoltage(analogRead(_sensorPin));
+      _pressure = convert_to_voltage( analogRead( _sensor_pin ) );
 
       if( _pressure < MIN_PACC )
       {
-          pumpOn();
+          pumpOn( );
       }
 
       if( _pressure > MAX_PACC )
       {
-          pumpOff();
+          pumpOff( );
       }
     }
 };
 
 
-
-
 // accumulator constructor
-Accumulator::Accumulator( byte sensorPin, byte controlPin )
+Accumulator::Accumulator( byte sensor_pin, byte control_pin )
 {
-  _sensorPin = sensorPin;
-  _controlPin = controlPin;
-
-
-  pinMode( _controlPin, OUTPUT ); // set pinmode to OUTPUT
+  _sensor_pin = sensor_pin;
+  _control_pin = control_pin;
 
+  pinMode( _control_pin, OUTPUT ); // set pinmode to OUTPUT
 
   // initialize pump to off
-  pumpOff();
+  pumpOff( );
 }
 
 
-
-
 // master Solenoid structure
 struct SMC {
     float _pressure1 = 0.0; // Initialize pressures to 0.0 to avoid false values
     float _pressure2 = 0.0;
-    byte _sensor1Pin = 99;
-    byte _sensor2Pin = 99;
-    byte _controlPin = 99;
+    byte _sensor1_pin = 99;
```

Comment:
Should we make 99 a DEFAULT_PIN of some kind?  I'm leaning towards int8_t for byte instead of char.  Do you have any preference?